godot-macros/src/class/derive_godot_class.rs

```diff
@@ -569,12 +569,20 @@ fn post_validate(base_ty: &Ident, is_tool: bool) -> ParseResult<()> {
     // b) a runtime check in class.rs > register_class_raw() and validate_class_constraints().
 
     let is_extension = is_class_virtual_extension(&base_ty.to_string());
+    let is_editor_plugin = is_class_editor_plugin(&base_ty.to_string());
```

Comment:
The naming of this variable is confusing, further up there is `is_editor_plugin` which has a different meaning (namely: whether `#[class(editor_plugin)]`) is specified.

Feel free to also rename `is_extension` if you find a scheme  that fits both better, and distinguishes them from the attribute keys.